packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -60,6 +67,8 @@ ReleaseNote.propTypes = {
   type: PropTypes.oneOf(['feature', 'enhancement', 'fix']).isRequired,
   topics: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
   body: PropTypes.string.isRequired,
+  bodyRenderer: PropTypes.oneOf(['mdx', 'excerpt']).isRequired,
```

Comment:
Hmm, let me push a commit...

Reply:
So: https://github.com/commercetools/commercetools-docs-kit/pull/375/commits/3d32a5669530177268f50ef765a9e93870fde48a

TL;DR: The parent should just format the body and pass it as a children.